backend/controllers/addGroup.js

```diff
@@ -22,7 +22,7 @@ const selectGroupFactory = (getUserAndGroups, saveGroup, maintainUrl, maintainTi
         maintainUrl,
         staff: { username: user.username, name: `${user.firstName} ${user.lastName}` },
         staffUrl,
-        groupDropdownValues,
+        groupDropdownValues: [{ text: '', value: '' }].concat(groupDropdownValues),
```

Comment:
need a default value here otherwise the select chooses the first one

Reply:
Not sure if you're interested or not but we've created a filter for default values, and it's used like: 

`      {{ govukSelect({
              name: 'attribute',
              id: 'attribute',
              label: {
                text: 'Select cell type',
                classes: 'govuk-visually-hidden'
              },
              items: cellAttributes | addDefaultSelectedVale('Select cell type', true)
          }) }}`

This is the filter code:
`  njkEnv.addFilter('addDefaultSelectedVale', (items, text, show) => {
    if (!items) return null
    const attributes = {}
    if (!show) attributes.hidden = ''

    return [
      {
        text,
        value: '',
        selected: true,
        attributes,
      },
      ...items,
    ]
  })`

Reply:
:+1: